### PR TITLE
XIVY-3449 Include target/classes by default in iar

### PR DESCRIPTION
**Pull request by alexsuter**
The guys who have modified `output` to `target/classes` in `.classpath`
in its projects will not encounter missing class files anymore in iar.

### CODE REVIEW

**Comment by alexsuter** [comment]
https://jenkins.ivyteam.io/job/project-build-plugin/job/feature%252FXIVY-3449-include-target-classes-by-default/

File: src/main/java/ch/ivyteam/ivy/maven/IarPackagingMojo.java [hunk 1]
```diff
@@ -47,7 +47,7 @@
 {
   public static final String GOAL = "pack-iar";
   private static final String[] DEFAULT_INCLUDES = new String[] {"**/*"};
-  private static final String[] DEFAULT_EXCLUDES = new String[] {"target", "target/**/*"};
+  private static final String[] DEFAULT_EXCLUDES = new String[] {"target", "%regex[target\\/(?!classes/).*]"};
```
Comment (ivy-rew) [inline src/main/java/ch/ivyteam/ivy/maven/IarPackagingMojo.java:50]: wow didn't know that regex statements are possible here. 
Thanks for this cool idea :)

**Comment by ivy-rew** [comment]
love to see that issue being solve: hopefully the PR is designed to be merged back to LTS as well -> origins from LTS8base.
... can you check that our demos no longer contain any pom workaround for this problem, thanks in advance.

**Comment by alexsuter** [comment]
I tried now with the help of @ivy-ohe the "proper" rebasing. Which we couldn't do on the command line, because:
- we did not know some commands
- I don't understand what we are exactly are doing.

For me its much easier cherry picking.

If we have to work this way, we should consider an agile uni. I think also some other developers in our team do not work that way.

**Comment by ivy-rew** [comment]
buuu I thought I have teached it already a hundred times. But yes a confluence page on that topic would be better anyway to conserve this way to solve issues for developers that will join the team in the future.

**Pull request closed without merging**